index.html

```diff
@@ -7,7 +7,6 @@
 <body>
 
 <h1>Hello Microverse</h1>
-<p>hello Microvese from Ahmed Hasan Rony</p>
```

Comment:
- [ ] As said in the project requirement, kindly make sure you add some style to your created element thanks its class in css

Reply:
I already added some css but with the heading tag but not with the class. Now I have added the class.